hathor/p2p/sync_v2/agent.py

```diff
@@ -308,16 +304,17 @@ def _run_sync(self) -> Generator[Any, Any, None]:
             self.log.debug('needed tx exist, sync transactions')
             self.update_synced(False)
             # TODO: find out whether we can sync transactions from this peer to speed things up
-            self.run_sync_transactions()
+            yield self.run_sync_transactions()
```

Comment:
**question:** don't we have to add `@inlineCallbacks` to this method?

Reply:
It already does, doesn't it?

Reply:
🤔 I probably misread it, nevermind. Thanks!